Technotes/AvoidFeedParsing.markdown

```diff
@@ -30,10 +30,10 @@ NetNewsWire also looks at the content of the feed. If it’s definitely an image
 
 Yes, this kind of thing happens in the real world: I’ve seen it. (Once I even saw a feed URL return a movie file.)
 
-We could more here, but it’s not often an issue, so it’s not a high priority. Just a good-to-have.
+We could do more here, but it’s not often an issue, so it’s not a high priority. Just a good-to-have.
 
 ## Thing It Never Does
 
 Feeds sometimes contain dates for modification times. NetNewsWire doesn’t trust these at all. In-feed dates are *never* used for making any decisions about parsing or not.
 
-When an article has a modification date, that date is stored in the database. But it’s there only in case it should be shown to the user. (Sometimes articles in a feed have a modification date but not a publication date — why oh why? — and in that case we display the modification date.)
\ No newline at end of file
+When an article has a modification date, that date is stored in the database. But it’s there only in case it should be shown to the user. (Sometimes articles in a feed have a modification date but not a publication date — why oh why? — and in that case we display the modification date.)
```

Comment:
Something changed here, too?

Reply:
Yeah, from the diff it seems so, but actually the sentence that looks added are also in the original file, I don't know why it looks as an insertion.

Reply:
Ah, I thought it may be a whitespace change.